src/components/FieldsetLegend/FieldsetLegend.module.css

```diff
@@ -0,0 +1,26 @@
+@import '../../design-tokens/mixins.css';
+
+/*------------------------------------*\
+    #FIELDSET LEGEND
+\*------------------------------------*/
+
+/**
+ * A label that's rendered as a <legend> for fieldsets.
+ * It contains the same characteristics as a label (ability to add flag for optional field, etc),
+ * but semantically/stylistically a bit different
+ * 1) Removes some default browser styles.
+ * 2) Adjust margin between legend and option list
+ */
+.legend {
```

Comment:
Should this class be `.fieldset-legend`?

Reply:
Yup, will change, thanks